kcidb/templates/misc.j2

```diff
@@ -13,11 +13,12 @@
 
 LEGEND
 
-    💥 ERROR    - Aborted. Test, tested code, or both might be faulty.
     ❌ FAIL     - Failed. Tested code is likely faulty.
+    💥 ERROR    - Aborted. Test, tested code, or both might be faulty.
+    ⚫ MISS     - Missing. Planned, but failed to execute.
```

Comment:
IMO we should use the 🚫  emoji in MISS, this can be more relevant as it suggests that something is missing.

Reply:
We can also think of "🟡" (yellow circle) or "🟠" (orange circle).

Reply:
Got it, Yeah then a black circle works fine.